src/IntegrationTests.jl

```diff
@@ -1,5 +1,159 @@
 module IntegrationTests
 
-# Write your package code here.
+using Pkg
+using PkgDependency
+
+export depending_projects
+
+"""
+    depending_projects(
+        package_name::AbstractString, 
+        package_filter::Union{TString,Regex} where {TString<:AbstractString}, 
+        project_tree::AbstractDict=PkgDependency.builddict(Pkg.project().uuid, Pkg.project())
+    ) -> Vector{String}
+
+Returns a list of packages that have the package `package_name` as a dependency. 
+
+# Arguments
+
+- `package_name`: Name of the dependency
+- `package_filter`: Ignore all packages that do not match `package_filter`. This includes the 
+        top node package of the graph. Child nodes always are checked for `package_name`, but 
+        they are not traveres if they do not match `package_filter`.
+- `project_tree`: Project tree, where to search the dependent packages. Each (sub-) package 
+        needs to be AbstractDict{String, AbstractDict}
+
+# Returns
+
+all packages which have the searched dependency
+
+"""
+function depending_projects(
+    package_name::AbstractString,
+    package_filter::Union{TString,Regex},
+    project_tree::AbstractDict=PkgDependency.builddict(Pkg.project().uuid, Pkg.project()),
+)::Vector{String} where {TString<:AbstractString}
```

Comment:
Should be equivalent to
```suggestion
function depending_projects(
    package_name::AbstractString,
    package_filter::Union{AbstractString,Regex},
    project_tree::AbstractDict=PkgDependency.builddict(Pkg.project().uuid, Pkg.project()),
)::Vector{String}
```